scripts/run_mypy_checks.py

```diff
@@ -769,37 +774,65 @@ def install_third_party_libraries(skip_install):
         install_third_party_libs.main()
 
 
-def get_mypy_cmd(files):
+def get_mypy_cmd(files, ci_check):
     """Return the appropriate command to be run.
 
     Args:
         files: list(list(str)). List having first element as list of string.
+        ci_check: bool. Whether the test is for CI.
 
     Returns:
         list(str). List of command line arguments.
     """
+    # TODO(#13113): Change mypy command to mypy path after Python3 migration.
```

Comment:
I've left a comment in teh TODO issue please update the issue with proper explanation!

Reply:
This TODO message is a little unclear. Maybe `Use path to mypy executable instead of "mypy" after Python 3 migration`

Also, does this only apply to the `mypy_cmd = 'mypy'` line? It seems like the code in the `else` block already uses the path. If it only applies to the block where `ci_check=True`, the TODO should be inside that block

Reply:
Merged two issues. Added #13113  to #13398 .